label_studio_converter/imports/yolo.py

```diff
@@ -15,6 +17,23 @@
 
 logger = logging.getLogger('root')
 
+def get_data(input_dir, img_exts):
+    get_labels = lambda files: list( filter(lambda fn: fn.endswith('.txt') and 'classes.txt' not in fn, files if type(files)==list else os.listdir(files)) )  
+    get_images = lambda files: list( filter(lambda fn: any([fn.endswith(img_ext) for img_ext in img_exts]), files if type(files)==list else os.listdir(files)) )  
+    images, labels = [], []
+    image_labels = {}
+    for dir_pth, dir_names, files in os.walk(input_dir):
+        if Path(dir_pth) == Path(input_dir):
+            continue         # skip input_dir. data should be at least one level in from input_dir
+        dir_imgs, dir_lbls = get_images( files ), get_labels( files )
+        if len(dir_imgs) > 0:
+            [images.append(f'{dir_pth}/{img}') for img in dir_imgs]
+        if len(dir_lbls) > 0:
+            [labels.append(f'{dir_pth}/{lbl}') for lbl in dir_lbls]
+    for image, label in zip(images, labels):
```

Comment:
There's no certainty that the `images` and `labels` lists will be in the same order, something like `for image, label in zip(sorted(images), sorted(labels))` does the trick.